test cases/failing/46 pkgconfig variables zero length value/meson.build

```diff
@@ -0,0 +1,16 @@
+project('variables-zero-length-test', 'c', version : '1.0')
```

Comment:
how would I go about this? it errors out on the first failed pkgconfig - how would I test for two things here?